server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -69,6 +73,9 @@ const (
 
 	// Exit code in Finished event indicating that the build was interrupted (i.e. killed by user).
 	InterruptedExitCode = 8
+
+	// How many results to return at a time for the cache scorecard.
+	scoreCardPageSize = 250
```

Comment:
Call this defaultScoreCardPageSize since the page size can be specified in request?

Reply:
Done